### PR TITLE
Add i18next localization files

### PR DESCRIPTION
**Pull request by Mixu78**
Adds English, Finnish and Swedish localization files to be used instead of hardcoded strings.

### CODE REVIEW

File: i18n/fi.json [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+          "<a href=\"https://github.com/slacka/WoeUSB\">WoeUSB</a> (Linux),",
+          "tai Boot Camp Assistant (macOS)."
+        ],
+        "missingPartitionTable": [
+          "Vaikuttaa siltä, että laite ei ole käynnistettävä\n\n",
+          "Laitteesta vaikuttaa puuttuvan osiotaulu,",
+          "ja voi olla tunnistamaton käynnistyksessä."
+        ],
+        "driveMissingPartitionTable": "Tämä levy ei vaikuta olevan käynnistettävä. Laitteesta vaikuttaa puuttuvan osiotaulu, ja voi olla tunnistamaton käynnistyksessä.",
+        "largeDriveSize": "Tämä on suuri laite! Varmista ettei levyllä ole mitään tarkeää!",
```
Comment (ihmemaassa) [inline i18n/fi.json]: "Tämä on suuri laite" olisko laite ennemmin levy, tikku tai asema

File: i18n/fi.json [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        "validation": [
+          "Levykuva kirjoitettiin onnistuneesti, mutta Abitikku havaitsi",
+          "korruptointiongemia tarkistaessa tikkua.",
+          "\n\nHarkitse toisen tikun käyttöä."
+        ],
+        "openSource": "Jokin meni pieleen avattaessa {{sourceName}}\n\nVirhe: {{errorMessage}}",
+        "flashFailure": "Jokin meni pieleen kirjoittaessa {{imageBasename}} kohteeseen $t(describeTarget, {\"count\": {{targets.length}}, \"desc\": \"{{targets.0.description}}\", \"name\": \"{{targets.0.displayName}}\"}).",
+        "driveUnplugged": [
+          "Abitikku ei saa yhteyttä tikkuun.",
+          "Irroititko tikun vahingossa irti?",
```
Comment (ihmemaassa) [inline i18n/fi.json]: "Irroititko tikun vahingossa?" tai "Otitko tikun vahingossa irti?"

File: i18n/fi.json [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+      "missingDrives": "Puuttuvia levyjä",
+      "invalidDrive": "Väärä levy: {{drive}}",
+      "invalidDrives": "Vääriä levyjä: {{drives}}",
+      "driveUnavailable": "Levy ei ole saatavilla: {{drive}}",
+      "driveWriteProtected": "Laite on kirjoitussuojattu",
+      "driveTooSmall": "Levy ei ole tarpeeksi suuri"
+    },
+    "menu": {
+      "mainLabel": "Näkymä",
+      "toggleDevTools": "Kytke kehittäjätila",
```
Comment (ihmemaassa) [inline i18n/fi.json]: "Vaihda kehittäjätilaan"

**Review by ihmemaassa — COMMENTED** [review]
englannissa ei mikään pistänyt silmään ja ruotsiin ei riitä osaaminen

**Comment by raikasdev** [comment]
If you translate by using a translator, use https://deepl.com

**Comment by DrVilepis** [comment]
As far as I am aware @Mixu78 can do all those languages fluently

File: i18n/en.json [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+        "describeTarget_other": "{{count}} targets"
+      },
+      "progress": {
+        "success_other": "Successful $t(messages.common.target, {\"count\": {{count}}})",
+        "failure_other": "Failed $t(messages.common.target, {\"count\": {{count}}})"
+      },
+      "info": {
+        "singleTarget": "to {{description}} ({{displayName}})",
+        "successfulTargets": "to {{successful}} $t(messages.common.target, {\"count\": {{successful}}})",
+        "failedTargets": "and failed to be flashed to {{failed}} $t(messages.common.target, {\"count\": {{failed}}})",
```
Comment (Esinko) [inline i18n/en.json]: Is `flashed` the correct word to use here? It's just a little uncommon. Would `written` work better?

File: i18n/en.json [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+        "header": "{{imageBasename}} was successfully flashed {{targets}}"
```
Comment (Esinko) [inline i18n/en.json:41]: Same thing about `flashed` (read above).

File: i18n/en.json [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+      },
+      "compatibility": {
+        "sizeNotRecommended": "Not recommended",
+        "tooSmall": "Too small",
+        "locked": "Locked",
+        "system": "System drive",
+        "containsImage": "Source drive",
```
Comment (Esinko) [inline i18n/en.json]: Should this be `Image source drive`?

File: i18n/en.json [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+        "largeDrive": "Large drive"
+      },
+      "warning": {
+        "tooSmall": "The selected source is {{bytes}} larger than this drive.",
```
Comment (Esinko) [inline i18n/en.json]: Maybe `The selected source image is larger than the selected drive`?
"This" might be a little abstract...

File: i18n/en.json [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+        "exitWhileFlashing": [
+          "You are currently flashing a drive.",
```
Comment (Esinko) [inline i18n/en.json:54]: Same thing about `flashing` (read above).

File: i18n/en.json [hunk 9]
```diff
@@ -0,0 +1,241 @@
...
+          "Closing Abitikku may leave your drive in an unusable state."
+        ],
+        "looksLikeWindowsImage": [
+          "It looks like you are trying to burn a Windows image.\n\n",
```
Comment (Esinko) [inline i18n/en.json]: Is it `burn` or `flash`? (Also note about the "flashed" word above)

File: i18n/en.json [hunk 10]
```diff
@@ -0,0 +1,241 @@
...
+          "Unlike other images, Windows images require special processing to be made bootable.",
```
Comment (Esinko) [inline i18n/en.json:59]: Would `... to make bootable` be better than `to be made bootable`?
(Might be totally wrong here)

File: i18n/en.json [hunk 11]
```diff
@@ -0,0 +1,241 @@
...
+          "We suggest you use a tool specially designed for this purpose, such as",
+          "<a href=\"https://rufus.akeo.ie\">Rufus</a> (Windows),",
+          "<a href=\"https://github.com/slacka/WoeUSB\">WoeUSB</a> (Linux),",
+          "or Boot Camp Assistant (macOS)."
+        ],
+        "missingPartitionTable": [
+          "It looks like this is not a bootable image.\n\n",
```
Comment (Esinko) [inline i18n/en.json:66]: Should we specify we are talking about a *disk* image?

File: i18n/en.json [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+          "The image does not appear to contain a partition table,",
+          "and might not be recognized or bootable by your device."
+        ],
+        "driveMissingPartitionTable": "It looks like this is not a bootable drive. The drive does not appear to contain a partition table, and might not be recognized or bootable by your device",
+        "largeDriveSize": "This is a large drive! Make sure it doesn't contain files that you want to keep.",
+        "systemDrive": "Selecting your system drive is dangerous and will erase your drive!",
```
Comment (Esinko) [inline i18n/en.json]: Erase what drive? We should be specific.

File: i18n/en.json [hunk 13]
```diff
@@ -0,0 +1,241 @@
...
+          "It looks like you are trying to burn a Windows image.\n\n",
+          "Unlike other images, Windows images require special processing to be made bootable.",
+          "We suggest you use a tool specially designed for this purpose, such as",
+          "<a href=\"https://rufus.akeo.ie\">Rufus</a> (Windows),",
+          "<a href=\"https://github.com/slacka/WoeUSB\">WoeUSB</a> (Linux),",
+          "or Boot Camp Assistant (macOS)."
+        ],
+        "missingPartitionTable": [
+          "It looks like this is not a bootable image.\n\n",
+          "The image does not appear to contain a partition table,",
```
Comment (Esinko) [inline i18n/en.json:67]: See previous comment.

File: i18n/en.json [hunk 14]
```diff
@@ -0,0 +1,241 @@
...
+          "and might not be recognized or bootable by your device."
+        ],
+        "driveMissingPartitionTable": "It looks like this is not a bootable drive. The drive does not appear to contain a partition table, and might not be recognized or bootable by your device",
+        "largeDriveSize": "This is a large drive! Make sure it doesn't contain files that you want to keep.",
+        "systemDrive": "Selecting your system drive is dangerous and will erase your drive!",
+        "sourceDrive": "Contains the image you chose to flash"
+      },
+      "error": {
+        "notEnoughSpaceInDrive": [
+          "Not enough space on the drive.",
```
Comment (Esinko) [inline i18n/en.json:77]: Would `There is not enough space on the drive` work better?

File: i18n/en.json [hunk 15]
```diff
@@ -0,0 +1,241 @@
...
+          "Please insert a larger one and try again."
+        ],
+        "genericFlashError": "Something went wrong. If it is a compressed image, please check that the archive is not corrupted.\n{{message}}",
```
Comment (Esinko) [inline i18n/en.json]: What is a compressed image? What image are we talking about?
That should be specified.

File: i18n/en.json [hunk 16]
```diff
@@ -0,0 +1,241 @@
...
+        "validation": [
+          "The write has been completed successfully but Abitikku detected potential",
```
Comment (Esinko) [inline i18n/en.json]: `write`, `flash` or `burn`? Mixing of terms feels unprofessional.

File: i18n/en.json [hunk 17]
```diff
@@ -0,0 +1,241 @@
...
+          "corruption issues when reading the image back from the drive.",
+          "\n\nPlease consider writing the image to a different drive."
+        ],
+        "openSource": "Something went wrong while opening {{sourceName}}\n\nError: {{errorMessage}}",
+        "flashFailure": "Something went wrong while writing {{imageBasename}} to $t(describeTarget, {\"count\": {{targets.length}}, \"desc\": \"{{targets.0.description}}\", \"name\": \"{{targets.0.displayName}}\"}).",
+        "driveUnplugged": [
+          "Looks like Abitikku lost access to the drive.",
+          "Did it get unplugged accidentally?",
+          "\n\nSometimes this error is caused by faulty readers that don't provide stable access to the drive."
```
Comment (Esinko) [inline i18n/en.json:91]: Should there be a comma after `Sometimes`?
I would personally write it like this: `This error can be sometimes caused by...`

File: i18n/en.json [hunk 18]
```diff
@@ -0,0 +1,241 @@
...
+        ],
+        "inputOutput": [
+          "Looks like Abitikku is not able to write to this location of the drive.",
```
Comment (Esinko) [inline i18n/en.json:94]: `of` or `on` the drive?

File: i18n/en.json [hunk 19]
```diff
@@ -0,0 +1,241 @@
...
+          "This error is usually caused by a faulty drive, reader, or port.",
+          "\n\nPlease try again with another drive, reader, or port."
+        ],
+        "childWriterDied": [
+          "The writer process ended unexpectedly.",
```
Comment (Esinko) [inline i18n/en.json:99]: `write`, `flash` or `burn`? Mixing of terms feels unprofessional.

File: i18n/en.json [hunk 20]
```diff
@@ -0,0 +1,241 @@
...
+          "Please try again, and contact the development team if the problem persists."
+        ],
+        "unsupportedProtocol": "Only http:// and https:// URLs are supported."
+      }
+    },
+    "errors": {
+      "EPERM": {
+        "title": "You're not authorized to perform this operation",
```
Comment (Esinko) [inline i18n/en.json:107]: `not authorized` -> `unauthorized`?

File: i18n/en.json [hunk 21]
```diff
@@ -0,0 +1,241 @@
...
+        "description": "Please ensure you have necessary permissions for this task"
```
Comment (Esinko) [inline i18n/en.json]: `necessary permissions` -> `the necessary permissions`?

File: i18n/en.json [hunk 22]
```diff
@@ -0,0 +1,241 @@
...
+      },
+      "EACCES": {
+        "title": "You don't have access to this resource",
+        "description": "Please ensure you have necessary permissions to access this resource"
```
Comment (Esinko) [inline i18n/en.json]: See comment above.

File: i18n/en.json [hunk 23]
```diff
@@ -0,0 +1,241 @@
...
+      },
+      "ENOMEM": {
+        "title": "Your system ran out of memory",
+        "description": "Please make sure your system has enough available memory for this task"
+      },
+      "errorCode": "Error code: {{code}}",
+      "genericError": "An error occurred"
+    },
+    "sudo": {
+      "privilegedAccessRequired": "Abitikku needs privileged access in order to flash disks.\n\nType your password to allow this."
```
Comment (Esinko) [inline i18n/en.json:122]: `write`, `flash` or `burn`? Mixing of terms feels unprofessional.

File: i18n/en.json [hunk 24]
```diff
@@ -0,0 +1,241 @@
...
+      "about": "About Abitikku"
+    },
+    "mainPage": {
+      "driveTitle": {
+        "noDescription": "Untitled Device",
+        "noTargets": "No targets found"
+      }
+    },
+    "flash": {
+      "success": "Flash complete!",
```
Comment (Esinko) [inline i18n/en.json:150]: `write`, `flash` or `burn`? Mixing of terms feels unprofessional.

File: i18n/en.json [hunk 25]
```diff
@@ -0,0 +1,241 @@
...
+      "failure": "Oops! Looks like the flash failed.",
```
Comment (Esinko) [inline i18n/en.json:151]: See comment above.

File: i18n/en.json [hunk 26]
```diff
@@ -0,0 +1,241 @@
...
+      "imageSelect": {
+        "osImages": "OS Images",
+        "all": "All"
+      },
+      "warningTitle": "Attention"
+    },
+    "progress-status": {
+      "starting": "Starting...",
+      "decompressing": "Decompressing...",
+      "flashing": "Flashing...",
```
Comment (Esinko) [inline i18n/en.json:168]: `write`, `flash` or `burn`? Mixing of terms feels unprofessional.

File: i18n/en.json [hunk 27]
```diff
@@ -0,0 +1,241 @@
...
+      "finishing": "Finishing...",
+      "validating": "Validating...",
+      "failed": "Failed"
+    },
+    "image-writer": {
+      "processEnded": {
+        "title": "The writer process ended unexpectedly",
```
Comment (Esinko) [inline i18n/en.json:175]: `writer`, `flasher` or `burner`? Mixing of terms feels unprofessional.

File: i18n/en.json [hunk 28]
```diff
@@ -0,0 +1,241 @@
...
+        "description": "Please try again, and contact the development team if the problem persists"
```
Comment (Esinko) [inline i18n/en.json:176]: `development team` -> `developer team`?

File: i18n/en.json [hunk 29]
```diff
@@ -0,0 +1,241 @@
...
+    },
+    "progress-button": {
+      "fallbackImageName": "Loading...",
+      "flash": "Flash!"
+    },
+    "flash-results": {
+      "descriptionLabel": "Target",
+      "deviceLabel": "Location",
+      "messageLabel": "Error",
+      "flashFail": "Flash failed!",
```
Comment (Esinko) [inline i18n/en.json:204]: `write`, `flash` or `burn`? Mixing of terms feels unprofessional.

File: i18n/en.json [hunk 30]
```diff
@@ -0,0 +1,241 @@
...
+      "flashSuccess": "Flash complete!",
```
Comment (Esinko) [inline i18n/en.json:205]: See above comment.

File: i18n/en.json [hunk 31]
```diff
@@ -0,0 +1,241 @@
...
+      "warningBig": "WARNING!"
+    },
+    "drive-selector": {
+      "nameLabel": "Name",
+      "sizeLabel": "Size",
+      "linkLabel": "Location",
+      "devicesAmount": "{{devices}} found",
+      "selectDevices": "Select {{selected}}",
+      "showHidden": "Show {{hidden}} hidden",
+      "systemDriveAlert": "Selecting your system drive is dangerous and will erase your drive!",
```
Comment (Esinko) [inline i18n/en.json]: Erase what drive? We should be specific.

**Review by Esinko — CHANGES_REQUESTED** [review]
Took a look at `en.json`. I'll read through the others later.

File: i18n/en.json (hunk 7 above)
Comment (developerfromjokela) [reply i18n/en.json]: This is refering to abitti image being larger than usb drive. Maybe `Abitti is larger than the selected drive` would be more suitable?

File: i18n/en.json (hunk 14 above)
Comment (developerfromjokela) [reply i18n/en.json:77]: I think keeping it short is better.

File: i18n/fi.json [hunk 32]
```diff
@@ -0,0 +1,243 @@
...
+    }
+  },
+  "shared": {
+    "permissions": {
+      "sudoError": {
+        "title": "Käyttäjällä ei ole tarvittavia oikeuksia",
+        "description": "Sovellus vaatii sudo-oikeuksia levyjen kirjoittamiseen"
+      },
+      "processDiedError": {
+        "title": "Korotettu prosessi kuoli yllättäen",
```
Comment (developerfromjokela) [inline i18n/fi.json]: Kuoli seems too scary for a consumer, maybe ended is better?

File: i18n/fi.json [hunk 33]
```diff
@@ -0,0 +1,243 @@
...
+        "description": "Prosessin virhekoodi on {{code}}"
+      },
+      "polkitError": {
+        "title": "Polkit tunnistautumisagenttia ei löytynyt",
+        "description": "Asenna polit autentikaatioagentti työpöytäympäristöösi jatkaakesi asennusta"
```
Comment (developerfromjokela) [inline i18n/fi.json]: autentikaatio or tunnistautumisagentti?

File: i18n/fi.json [hunk 34]
```diff
@@ -0,0 +1,243 @@
...
+        "largeDriveSize": "Tämä on suuri levy! Varmista ettei levyllä ole mitään tarkeää!",
+        "systemDrive": "Järjestelmälevyn valitseminen on vaarallista ja voi tehdä järjestelmästä käyttäkelvottoman!",
+        "sourceDrive": "Sisältää levykuvan jota yrität kirjoittaa"
+      },
+      "error": {
+        "notEnoughSpaceInDrive": [
+          "Ei riittävästi tallennustilaa tikulla",
+          "Liitä isompi tikku ja yritä uudelleen"
+        ],
+        "genericFlashError": "Jokin meni pieleen. Jos kyseessä on puristettu levykuva, tarkista ettei se ole korruptoitunut.\n{{message}}",
```
Comment (developerfromjokela) [inline i18n/fi.json]: Olisiko `Jos kyseessä on puristettu levykuva, tarkista ettei se ole korruptoitunut.` kokonaan pois, sillä abitikkuun ei voi lisätä ulkopuolisia levykuvia?

File: i18n/fi.json [hunk 35]
```diff
@@ -0,0 +1,243 @@
...
+        "validation": [
+          "Levykuva kirjoitettiin onnistuneesti, mutta Abitikku havaitsi",
```
Comment (developerfromjokela) [inline i18n/fi.json]: ~`Levykuva`~ `Abitti`

File: i18n/fi.json [hunk 36]
```diff
@@ -0,0 +1,243 @@
...
+      "EPERM": {
+        "title": "Sinulla ei ole oikeuksia suorittaa tätä tehtävää",
+        "description": "Varmista tarvittavat oikeudet tämän tehtävän suorittamiseen"
+      },
+      "EACCES": {
+        "title": "Sinulla ei ole pääsyä tähän resurssiin",
+        "description": "Varmista tarvittavat oikeudet resurssiin ja yritä uudelleen"
+      },
+      "ENOMEM": {
+        "title": "Järjestelästäsi on loppunut muisti",
```
Comment (developerfromjokela) [inline i18n/fi.json]: Typo, Järjestelmästäsi

File: i18n/fi.json [hunk 37]
```diff
@@ -0,0 +1,243 @@
...
+    }
+  },
+  "gui": {
+    "store": {
+      "missingDrive": "Puuttuva levy",
+      "missingDrives": "Puuttuvia levyjä",
+      "invalidDrive": "Väärä levy: {{drive}}",
+      "invalidDrives": "Vääriä levyjä: {{drives}}",
+      "driveUnavailable": "Levy ei ole saatavilla: {{drive}}",
+      "driveWriteProtected": "Laite on kirjoitussuojattu",
```
Comment (developerfromjokela) [inline i18n/fi.json]: Laite -> Levy

**Review by developerfromjokela — CHANGES_REQUESTED** [review]
Checked English, and Finnish. Jag talar inte svenska, so someone else should have a look on Swedish translation.

File: i18n/en.json (hunk 20 above)
Comment (Mixu78) [reply i18n/en.json:107]: "Not authorized" feels better, I'll leave this as it is

File: i18n/en.json (hunk 28 above)
Comment (Mixu78) [reply i18n/en.json:176]: In my opinion the terms work like this:

`Development team`: Team that develops the application
`Developer team`: A team that consists of developers

Thus `Development team` fits better.

File: i18n/en.json (hunk 4 above)
Comment (Mixu78) [reply i18n/en.json]: Flashed is correct for this, and the original Etcher uses these words so I think this should be left as it is

File: i18n/en.json (hunk 9 above)
Comment (Mixu78) [reply i18n/en.json]: This doesn't really matter as abitikku should never aquire and try to flash a windows image, but I'll change it anyway

File: i18n/en.json (hunk 10 above)
Comment (Mixu78) [reply i18n/en.json:59]: `to be made bootable` is correct

File: i18n/en.json (hunk 11 above)
Comment (Mixu78) [reply i18n/en.json:66]: This message should never appear, like the `looksLikeWindowsImage` message. I'll leave it as it is

File: i18n/en.json (hunk 15 above)
Comment (Mixu78) [reply i18n/en.json]: This could be shortened to "Something went wrong" as abitikku only flashes abitti images.

File: i18n/en.json (hunk 19 above)
Comment (Mixu78) [reply i18n/en.json:99]: I feel like `flash` is the process of writing the image data to the drive and thus "writer process" is fine

**Review by ahnl — APPROVED** [review]
Käännös on hyvä, kiitos kontribuutiosta! Minusta tämän voi yhdistää ihan tälläisenä.

### Ehdotuksen sana
Kannattaa pitää tässä mielessä myös se, että käyttäjäkunta saattaa olla tietokoneosaamiseltaan melko alhaista. Käännöksiin voi siis käyttää mahdollisimman simppeleitä käsitteitä. Ohjelmaa ei käytetä kuin Abitti-tikkujen luomiseen, jolloin voidaan yrittää kulkea käyttäjän kanssa mahdollisimman käsi kädessä luomisprosessin kanssa.

Esimerkiksi:
```
"largeDriveSize": "Tämä on suuri levy! Varmista ettei levyllä ole mitään tarkeää!"
```
```
"largeDriveSize": "Tikulla on paljon tavaraa! Varmista ettei tikulla ole mitään tarkeää!"
```

```      "progress": {
         "success_one": "Onnistunut kohde",
         "success_other": "Onnistuneet kohteet",
         "failure_one": "Epäonnistunut kohde",
         "failure_other": "Epäonnistuneet kohteet"
       },
```

```      "progress": {
         "success_one": "Onnistunut tikku",
         "success_other": "Onnistuneet tikut",
         "failure_one": "Epäonnistunut tikku",
         "failure_other": "Epäonnistuneet tikkut"
       },
```

Tämän muutoksen voi tehdä halutessaan, mutta se ei ole mikään välttämätön.

**Review by ahnl — APPROVED** [review]

**Comment by ahnl** [comment]
Moving over to #4

**Pull request closed without merging**